modules/owncloud_web/pages/web_for_users.adoc

```diff
@@ -65,18 +65,27 @@ A click on the three vertical dots opens an actions menu. Actions can be perform
 
 === Spaces
 
-Spaces are organizational units to sort content and make it available to certain members.
+Spaces are special folders for teams. They are organized by the team members itself so that admins don't need to manage them. Even if members leave, the files stay in place so your team can keep on working.
```

Comment:
```suggestion
Spaces are special folders for teams. They are organized by the team members themselves so that admins don't need to manage them. Even if members leave, the files stay in place so your team can keep on working.
```